Content.Client/UserInterface/GameHud.cs

```diff
@@ -47,6 +47,8 @@ public interface IGameHud
         Action<bool> SandboxButtonToggled { get; set; }
 
         Control HandsContainer { get; }
+        Action<string> OnHandChanged { get; set; }
```

Comment:
Could you just invoke the eventbus directly instead? That way you can also remove the handssystem too.